storybook/components/DocsContainer/ComponentDocsTemplate/ComponentDocsTemplate.tsx

```diff
@@ -41,7 +40,6 @@ export const ComponentDocsTemplate = ({
       <Content>
         <Title context={context} />
         <Links context={context} />
-        <KaioInstallation context={context} />
```

Comment:
OOS - We should replace usage of this template with MDX and then remove this instead

Reply:
Will create a ticket after this work 👍